app/src/main/java/com/it3048/mobiledeviceproject/AddMeetingForm.kt

```diff
@@ -20,13 +20,11 @@ class AddMeetingForm: DialogFragment() {
 
         rootView.cancelButton.setOnClickListener{
             dismiss()
-            rootView.editTextMeetingTitle.setText("")
-            rootView.meetingColor.selectedItem.toString()
-            rootView.meetingDate.setText("")
-            rootView.editTextDescripton.setText("")
-            rootView.editTextLocLink.setText("")
+            clearFields(rootView)
```

Comment:
Took the code to clear fields and put it into a function, added the function to the submit button too so that when a 2nd submission is made it starts with cleared fields.